components/base/BaseCollectionForm.vue

```diff
@@ -43,32 +43,55 @@
   </div>
 </template>
 
-<script lang="ts">
-import { Component, Prop, PropSync, Ref, Vue } from 'nuxt-property-decorator'
+<script setup lang="ts">
+import { Ref, ref, toRefs } from 'vue'
 import { NeoField } from '@kodadot1/brick'
+import Auth from '@/components/shared/Auth.vue'
+import MetadataUpload from '@/components/shared/DropUpload.vue'
+import BasicInput from '@/components/shared/form/BasicInput.vue'
 
-const components = {
-  Auth: () => import('@/components/shared/Auth.vue'),
-  MetadataUpload: () => import('@/components/shared/DropUpload.vue'),
-  BasicInput: () => import('@/components/shared/form/BasicInput.vue'),
-  NeoField,
-}
+const props = defineProps({
+  label: {
+    type: String,
+    default: 'mint.collection.create',
+  },
+  protectiveMargin: Boolean,
+  name: String,
+  description: String,
+  file: Blob,
+})
+const emit = defineEmits(['update:name', 'update:description', 'update:file'])
+
+const vName = computed({
+  get: () => props.name,
+  set: (value) => {
+    emit('update:name', value)
+  },
+})
+
+const vDescription = computed({
+  get: () => props.description,
+  set: (value) => {
+    emit('update:description', value)
+  },
+})
```

Comment:
Maybe this would be of use

https://vueuse.org/core/useVModel/

Reply:
@daiagi thanks for the link. Would you want me to use that instead?
Also there's a defineModel macro on vue 3.3 but i believe is not available here in bridge

Reply:
I used useVModel now @daiagi Thanks for the link.

Reply:
Really? I don't see that you use it.

Reply:
Oh, applied it on the other branch, and all my other reactors. I might have overlooked this branch. Will do it in a while, just driving now

Reply:
> just driving now

be safe

Reply:
So traffic here in ph 
<img width="745" alt="image" src="https://github.com/kodadot/nft-gallery/assets/3810177/7d39c7de-3551-494e-ae4e-30aeab034300">
Managed to edit and push it now while driving
<img width="419" alt="image" src="https://github.com/kodadot/nft-gallery/assets/3810177/a639b43b-0065-4ce2-b402-ef104d9cc908">
LOL